streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordQueue.java

```diff
@@ -55,7 +55,6 @@ public class RecordQueue {
 
     private final Sensor droppedRecordsSensor;
     private final Sensor consumedSensor;
-    private long totalBytesBuffered;
     private long headRecordSizeInBytes;
```

Comment:
I did keep this one, because there was a follow up PR that depends on it, and I did not want to revert (or change) the other one.

https://github.com/apache/kafka/commit/a6c5a74fdbdce9a992b47706913c920902cda28c